docs/source/technical/vigasp.md

```diff
@@ -0,0 +1,20 @@
+# Overview: 
+VIGAS/P stands for Veteriary Institute Genome Analysis System/Platform. VIGASP uses IRIDA for data management, data access control, analysis and storing the results. IRIDA uses Galaxy to execute the pipelines. 
+
+# Accessing VIGAS and logging in
+VIGAS is accessible only from official VI Laptops or through VI-VPN. 
+Those who want to get user names has to send email to Jeevan (jeevan.karloss.antony-samy@vetinst.no) with following details: 
+1. Explanation: Why do you need access to VIGAS
+2. Preferred user name and mobile number
+3. List of species data do you want to have access for. We wil talk to the managers of those projects in VIGAS and give you access to those data/metadata/results.
```

Comment:
You need to add that the person will have to meet the bioinf team in person and to show a NVI ID card due to the confidentiality thing.